services-js/registry-certs/client/birth/components/VerifyIdentificationComponent.tsx

```diff
@@ -13,11 +14,17 @@ import FieldsetComponent from '../components/FieldsetComponent';
 import SupportingDocumentsInput from './SupportingDocumentsInput';
 import IdIcon from '../icons/IdIcon';
 
+import UploadableFile from '../../models/UploadableFile';
+import { BirthCertificateRequestInformation } from '../../types';
+
 import { SECTION_HEADING_STYLING, SUPPORTING_TEXT_STYLING } from '../styling';
 
 interface Props {
+  requestInformation: BirthCertificateRequestInformation;
   sectionsToDisplay?: 'all' | 'supportingDocumentsOnly';
-  updateSupportingDocuments: (documents: File[]) => void;
+  uploadSessionId: string;
+  supportingDocuments?: UploadableFile[];
+  updateSupportingDocuments: (documents: UploadableFile[]) => void;
```

Comment:
I would expect that `uploadSessionId` and `supportingDocuments` would come off of the `requestInformation` prop. I assume this is a vestige of when this component would have to work w/o a `BirthCertificateRequestInformation` object?

I think it should be cleaned up, but it can be done later.